plugins/toolkit/jetbrains-core/it/software/aws/toolkits/jetbrains/services/lambda/java/JavaLocalLambdaRunConfigurationIntegrationTest.kt

```diff
@@ -49,20 +51,17 @@ class JavaLocalLambdaRunConfigurationIntegrationTest(private val runtime: Lambda
     @JvmField
     val projectRule = HeavyJavaCodeInsightTestFixtureRule()
 
-    @Rule
-    @JvmField
-    val temporaryFolder = TemporaryFolder()
-
     private val mockId = "MockCredsId"
     private val mockCreds = AwsBasicCredentials.create("Access", "ItsASecret")
     private val input = RuleUtils.randomName()
+    private val customGradleUserHome = File(System.getProperty("java.io.tmpdir"), "gradle-user-home").apply {
+        mkdirs()
+    }
 
     @Before
     fun setUp() {
         setSamExecutableFromEnvironment()
-
-        val gradleUserHome = temporaryFolder.newFolder("gradle-user-home")
-        System.setProperty("gradle.user.home", gradleUserHome.absolutePath)
+        System.setProperty("gradle.user.home", customGradleUserHome.absolutePath)
```

Comment:
as discussed offline, this sets the property for the current running gradle process, which in effect does not do anything.

this needs to be set directly on the daemons of test projects being invoked, either through a `gradle.properties` file, or by somehow passing it as an env var through SAM CLI

Reply:
working on separate [PR] (https://github.com/aws/aws-toolkit-jetbrains/pull/4549)